frontend/src/features/admin-form/create/workflow/components/WorkflowContent/StepLabel.tsx

```diff
@@ -1,26 +1,10 @@
-import {
-  Flex,
-  Icon,
-  PlacementWithLogical,
-  Stack,
-  Text,
-  useBreakpointValue,
-} from '@chakra-ui/react'
-
-import { BxsInfoCircle } from '~assets/icons'
-import Tooltip from '~components/Tooltip'
+import { Flex, Stack, Text } from '@chakra-ui/react'
 
 type StepLabelProps = {
   stepNumber: number
-  tooltipLabel?: string
 }
 
-export const StepLabel = ({ stepNumber, tooltipLabel }: StepLabelProps) => {
```

Comment:
remove `tooltipLabel` based on YAGNI principle, can re-add if needed in the future.